tests/fixtures/nodes/onoff-switch-example-app.json

```diff
@@ -0,0 +1,771 @@
+        },
+        "lastConnectErrorValue": null,
+        "generatedCommandList": [1, 5, 7],
+        "acceptedCommandList": [0, 2, 4, 6, 8],
+        "attributeList": [
+          0, 1, 2, 3, 4, 5, 6, 7, 65528, 65529, 65531, 65532, 65533
+        ],
+        "featureMap": 1,
+        "clusterRevision": 1,
+        "_type": "chip.clusters.Objects.NetworkCommissioning"
+      },
+      "DiagnosticLogs": {
+        "generatedCommandList": [1],
+        "acceptedCommandList": [0],
+        "attributeList": [65528, 65529, 65531, 65532, 65533],
+        "featureMap": 0,
+        "clusterRevision": 1,
+        "_type": "chip.clusters.Objects.DiagnosticLogs"
+      },
+      "GeneralDiagnostics": {
+        "networkInterfaces": [
+          {
+            "name": "WIFI_AP_DEF",
+            "isOperational": true,
+            "offPremiseServicesReachableIPv4": null,
+            "offPremiseServicesReachableIPv6": null,
+            "hardwareAddress": {
+              "_type": "bytes",
+              "value": "AAAAAAAA"
+            },
+            "IPv4Addresses": [],
+            "IPv6Addresses": [],
+            "type": 1,
+            "_type": "chip.clusters.Objects.GeneralDiagnostics.Structs.NetworkInterfaceType"
+          },
+          {
+            "name": "WIFI_STA_DEF",
+            "isOperational": true,
+            "offPremiseServicesReachableIPv4": null,
+            "offPremiseServicesReachableIPv6": null,
+            "hardwareAddress": {
+              "_type": "bytes",
+              "value": "hPcDJ8rI"
+            },
+            "IPv4Addresses": [],
+            "IPv6Addresses": [],
+            "type": 1,
+            "_type": "chip.clusters.Objects.GeneralDiagnostics.Structs.NetworkInterfaceType"
+          }
+        ],
+        "rebootCount": 12,
+        "upTime": 458,
+        "totalOperationalHours": 0,
+        "bootReasons": 1,
+        "activeHardwareFaults": [],
+        "activeRadioFaults": [],
+        "activeNetworkFaults": [],
+        "testEventTriggersEnabled": false,
+        "generatedCommandList": [],
+        "acceptedCommandList": [0],
+        "attributeList": [
+          0, 1, 2, 3, 4, 5, 6, 7, 8, 65528, 65529, 65531, 65532, 65533
+        ],
+        "featureMap": 0,
+        "clusterRevision": 1,
+        "_type": "chip.clusters.Objects.GeneralDiagnostics"
+      },
+      "SoftwareDiagnostics": {
+        "threadMetrics": [],
+        "currentHeapFree": 116140,
+        "currentHeapUsed": 138932,
+        "currentHeapHighWatermark": 153796,
+        "generatedCommandList": [],
+        "acceptedCommandList": [0],
+        "attributeList": [0, 1, 2, 3, 65528, 65529, 65531, 65532, 65533],
+        "featureMap": 0,
+        "clusterRevision": 1,
+        "_type": "chip.clusters.Objects.SoftwareDiagnostics"
+      },
+      "ThreadNetworkDiagnostics": {
+        "TLVValue": {
+          "0": {
+            "TLVValue": null,
+            "Reason": "InteractionModelError: Failure (0x1)"
```

Comment:
I guess we can drop all of `ThreadNetworkDiagnostics` since it all fails to decode anyway.